termwiz/README.md

```diff
@@ -12,6 +12,7 @@ Included functionality:
 * `Surface` models a terminal display and its component `Cell`s
 * Terminal attributes are aware of modern features such as
   True Color, [Hyperlinks](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda)
+  True Color, [Hyperfiles](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda)
```

Comment:
`True Color` is a separate feature already mentioned above

Reply:
Here I can delete this too. I

Reply:
done